src/components/activityEventElems/RedeemEventElem.tsx

```diff
@@ -75,10 +73,16 @@ export default function RedeemEventElem({
         </div>
       }
       extra={
-        <div style={{ color: colors.text.secondary }}>
+        <div className="text-grey-900 dark:text-slate-100">
           <Trans>
             <ETHAmount amount={event.returnAmount} /> overflow received
           </Trans>
+          {event.memo && (
+            <>
+              <br />
```

Comment:
**suggestion**: could use a `mt-5` or similar class on the RichNote below instead, wdyt?

Reply:
great call